README.md

```diff
@@ -855,6 +856,17 @@ value1: or('source1', 'source2', 'source3'), // true
 value2: or(not('source1'), 'source2', not('source3')) // false
 ```
 
+##### `pad`
+adds zero-padding to a number and returns a string, allows composing
+
+```js
+number1: 1,
+string1: '3',
+string2: '5',
+example: pad('number1','string1'), // 001
+example2: pad(raw(8),'string2') // 00001
```

Comment:
This should be `//00008` right?

Same typo exists in the PR description.

Reply:
Yep, missed that.  Sorry!